README.md

```diff
@@ -1,91 +1,82 @@
+# SwarmDB
+
 [![Build Status](https://travis-ci.org/bluzelle/swarmDB.svg?branch=devel)](https://travis-ci.org/bluzelle/swarmDB)
+[![License](https://img.shields.io/:license-AGPLv3-blue.svg?style=flat-square)](https://github.com/bluzelle/swarmdb/blob/master/LICENSE)
+[![Twitter](https://img.shields.io/badge/twitter-@bluzelle-blue.svg?style=flat-square)](https://twitter.com/bluzelle)
+[![Slack chat](https://img.shields.io/badge/slackchat-join%20chat-brightgreen.svg?style=flat-square)](https://bluzelle-swarm.slack.com)
 
-BUILD INSTRUCTIONS
-==================
+## ABOUT SWARMDB
 
-macOS BOOST Installation 
--
-Bluzelle daemon uses the boost libraries, notably the Beast Library by Vinnie Falco, so we require Boost version 1.66.0, 
-released on December 18th, 2017.    
+Bluzelle brings together the sharing economy and token economy. Bluzelle enables people to rent out their computer storage space to earn a token while dApp developers pay with a token to have their data stored and managed in the most efficient way.
 
-NOTE: This will overwrite an existing Boost installation if it exists, if you have installed Boost with 
-brew or some other package manager, you may wish to uninstall the older version of Boost first.
+## Getting Started
 
-So open up a console and get started by downloading boost and building:
-```
-wget -c http://sourceforge.net/projects/boost/files/boost/1.66.0/boost_1_66_0.tar.bz2/download
-tar jxvf download
-rm download
-cd boost_1_66_0
-./bootstrap.sh 
-./b2 toolset=darwin install
-```
+If you want to deploy your swarm immediately you can use our docker-compose quickstart instructions:
 
-Feel free to use the "-j N" option with b2 to speed things up a bit -- you would need to determine how many CPU cores are on your machine. Basically, if you want to use multiple CPU cores, specify that number here in place of N. 
+Docker Swarm Deploy
 
-Now you will have the Boost 1.66.0 libraries installed in 
+1. Download sample [peerlist.json](https://gist.github.com/amastracci/b6144d32bd790cf1a93e3c498a9d46cc)
+2. Add your swarm machines' IP to the "host" field of each peer in peerlist.json
+3. Host your peerlist.json at a publically accessable URL
```

Comment:
Do we want to make a note that the file must be hosted in a non HTTPS and non redirecting URL?

Reply:
updated